core/java/iesi-core/src/main/java/io/metadew/iesi/component/http/HttpComponentService.java

```diff
@@ -90,6 +118,30 @@ public HttpComponent convert(HttpComponentDefinition httpComponentDefinition,
                         .map(queryParameter -> HttpQueryParameterService.getInstance().convert(queryParameter, actionExecution))
                         .collect(Collectors.toList())
         );
+
+
+        UUID uuid = UUID.randomUUID();
+        HttpComponentTrace httpComponentTrace = new HttpComponentTrace(
+                new ComponentTraceKey(uuid),
+                actionExecution.getExecutionControl().getRunId(),
+                actionExecution.getExecutionControl().getProcessId(),
+                actionParameterName,
+                COMPONENT_TYPE,
+                httpComponent.getReferenceName(),
+                httpComponent.getDescription(),
+                httpComponent.getVersion(),
+                httpComponent.getHttpConnection().getReferenceName(),
+                httpComponent.getType(),
+                httpComponent.getEndpoint(),
+                httpComponent.getHeaders().stream().map(header -> convertHeaders(header, uuid))
+                        .collect(Collectors.toList()),
+                httpComponent.getQueryParameters().stream().map(queries -> convertQueries(queries, uuid))
+                        .collect(Collectors.toList())
+        );
```

Comment:
could this be exported to a dedicated class 'HttpComponentTraceService' with method 'public HttpComponentTrace convert(HttpComponent httpComponent, ActionExecution actionExecution, String actionParameterName);